docs/openstack/glance_adoption.md

```diff
@@ -88,11 +94,6 @@ spec:
       storageClass: "local-storage"
       storageRequest: 10G
       glanceAPIInternal:
-        externalEndpoints:
-        - endpoint: internal
-          ipAddressPool: internalapi
-          loadBalancerIPs:
-          - 172.17.0.80
```

Comment:
to match the old example you may also want to have the override for ceph?

Reply:
could you please give me an example?
I didn't find what is the proper way to override ceph

Reply:
sorry, maybe what I wrote was confusing. I meant you removed the `externalEndpoints` in this `Using Ceph storage backend` section. you might want to add the service override for this, like above for the `glanceAPIInternal`?

```
        override:
          service:
            metadata:
              annotations:
                metallb.universe.tf/address-pool: internalapi
                metallb.universe.tf/allow-shared-ip: internalapi
                metallb.universe.tf/loadBalancerIPs: 172.17.0.80
            spec:
              type: LoadBalancer
```